packages/cloud-cognitive/src/components/Toolbar/ToolbarButton.js

```diff
@@ -6,52 +6,134 @@
  */
 
 import { Button } from 'carbon-components-react';
+
+import {
+  Popover,
+  PopoverContent,
+} from 'carbon-components-react/es/components/Popover';
```

Comment:
Is this the way unstable components from Carbon are meant to be used? Direct imports from /es/ or /lib/ can be problematic, because our users need both. An import from /es/ will probably work in Node environments which load ESM, eg our jest tests, but may not survive a WebPack bundling which loads the /lib/ transpiled versions...

Reply:
You're absolutely right, this won't play nice with CommonJS. When I started looking at this, `Popover` was unavailable from the entry point but luckily this has changed with https://github.com/carbon-design-system/carbon/pull/9965 — I'll update to use this instead.

If we come across this scenario again, there's the option of promoting https://github.com/carbon-design-system/ibm-cloud-cognitive/pull/1402 into `babel-preset-ibm-cloud-cognitive` — this ensures ES module imports don't make it into the CommonJS build and vice-versa.